<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todolist PySide2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="todoView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="todoedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>381</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>94</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>94</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Todo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="completeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>94</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>